web/components/bet-panel.tsx

```diff
@@ -179,12 +178,7 @@ export function BuyPanel(props: {
   const initialProb = getProbability(contract)
   const isPseudoNumeric = contract.outcomeType === 'PSEUDO_NUMERIC'
 
-  const windowSize = useWindowSize()
-  const initialOutcome =
-    windowSize.width && windowSize.width >= 1280 ? 'YES' : undefined
```

Comment:
This code looked like it was doing something, but it wasn't, because `useWindowSize` always starts out returning `undefined` dimensions until its internal `useEffect` runs, so `windowSize.width` was always falsy and `initialOutcome` was always `undefined`.